docs/src/pages/[platform]/connected-components/storage/storagemanager/react.mdx

```diff
@@ -86,7 +88,7 @@ A resumable upload will upload the file in chunks. This allows users to pause an
 
 You might want to process or modify the file(s) and/or file name(s) before they are uploaded. One common situation is you may want to ensure files uploaded are at unique keys by hashing the file contents and using that as the key rather than the filename.
 
-You can pass a `processFile` function to the StorageManager which accepts an object with `file: [File](https://developer.mozilla.org/en-US/docs/Web/API/File)` and `key: string` and should return an object with file, key, and any other Storage configurations. The `processFile` can either return synchronously or return a Promise. This example uses a Promise to read the contents of the file and create a hash for the key.
+You can pass a `processFile` function to the StorageManager which accepts an object with `file`: [File](https://developer.mozilla.org/en-US/docs/Web/API/File), and `key`: string, and should return an object with file, key, and any other Storage configurations. The `processFile` can either return synchronously or return a Promise. This example uses a Promise to read the contents of the file and create a hash for the key.
```

Comment:
Not related to the changes. 
```
Other uses-cases for processing the file before upload:
```
The numbered list is should be 1, 2?

Reply:
In markdown you can just do 1. 1. 1. and it will auto-number it:

1.
1.
1.